godump_test.go

```diff
@@ -891,6 +891,20 @@ func TestDumpWithCustomWriter(t *testing.T) {
 	}
 }
 
+func TestDumpWithCustomSkipStackFrames(t *testing.T) {
+	// caller stack frames are
+	//	1	godump.go:296              github.com/goforj/godump.findFirstNonInternalFrame			skip by initialCallerSkip
+	//	2	godump.go:273              github.com/goforj/godump.printDumpHeader						skip by initialCallerSkip
+	//	3	godump.go:185              github.com/goforj/godump.(*Dumper).DumpStr					skip by function names contain godump
+	//	4	godump_test.go:899         github.com/goforj/godump.TestDumpWithCustomSkipStackFrames	skip by function names contain godump
+	//	5	testing.go:1792            testing.tRunner
+	out := NewDumper().DumpStr("test")
+	assert.Contains(t, out, "testing.go")
```

Comment:
This shows the current detection has issue

The stack should return godump_test.go, not testing.go

I'm suggesting a fix

https://github.com/goforj/godump/pull/31#discussion_r2181503810

So I feel like a fix could also help with the fact I wanted to test wrapper outside the stack.